release/ray_release/test_automation/state_machine.py

```diff
@@ -1,13 +1,21 @@
+from datetime import datetime, timedelta
+
 import boto3
 from github import Github
+from pybuildkite.buildkite import Buildkite
 
 from ray_release.test import (
     Test,
     TestState,
 )
+from ray_release.logger import logger
 
 RAY_REPO = "ray-project/ray"
 AWS_SECRET_GITHUB = "ray_ci_github_token"
+AWS_SECRET_BUILDKITE = "ray_ci_buildkite_token"
+MAX_BISECT_PER_DAY = 10
```

Comment:
is this per test? or for all tests? could you add a comment that explains?

Reply:
reading the code, seems that this is for all tests. is 10 going to be enough?

Reply:
Right this is for all (release) tests

Often when there are more than 10 test failures a day, it tends to be a mass infra issue. Let's start a bit conservative and adjust if needed.